ctlra/devices/ni_kontrol_x1_mk2.c

```diff
@@ -510,9 +605,23 @@ ctlra_ni_kontrol_x1_mk2_connect(ctlra_event_func event_func,
 	dev->base.light_set = ni_kontrol_x1_mk2_light_set;
 	dev->base.light_flush = ni_kontrol_x1_mk2_light_flush;
 	dev->base.usb_read_cb = ni_kontrol_x1_mk2_usb_read_cb;
+	dev->base.feedback_digits = ni_kontrol_x1_mk2_feedback_digits;
 
 	dev->base.event_func = event_func;
 	dev->base.event_func_userdata = userdata;
+#if 1
```

Comment:
Looks like you don't need the preprocessor foo :)

Reply:
Fixed, thanks!